starter-code/server.js

```diff
@@ -35,8 +40,16 @@ app.get('/articles', function(request, response) {
 
 app.post('/articles', function(request, response) {
   client.query(
-    '', // TODO: Write a SQL query to insert a new author, ON CONFLICT DO NOTHING
-    [], // TODO: Add the author and "authorUrl" as data for the SQL query
+    // DONE: Write a SQL query to insert a new author, ON CONFLICT DO NOTHING
+    // estimate 5 min actual 2 min
+    `INSERT INTO authors (author, "authorUrl")
+    VALUES ($1 and $2) ON CONFLICT DO NOTHING;`,
```

Comment:
really close. This query gives an error. Take a look at your values of the inserts we did in the class 9 demo and see if you can find the difference in syntax.